### PR TITLE
refactor(UI): :recycle: Refactor and improve editableLabel component

### PR DESCRIPTION
**Pull request by RainingComputers**

### CODE REVIEW

File: meiki_ui/src/components/EditableLabel.svelte [hunk 1]
```diff
@@ -1,36 +1,53 @@
 <script lang="ts">
-    // TODO redesign this component
+    // Stolen and improved upon https://svelte.dev/repl/29c1026dda3c47a187bd21afa0782df1?version=3.48.0
```
Comment (alexsanjoseph) [inline meiki_ui/src/components/EditableLabel.svelte:2]: Brilliant!

File: meiki_ui/src/components/app/toolbar/AppToolbar.svelte [hunk 2]
```diff
@@ -12,8 +12,9 @@
     import UserIcon from "$cmp/icons/UserIcon.svelte"
     import ToolBarAction from "$cmp/toolbar/Action.svelte"
     import ToolbarSyncIndicator from "$cmp/app/toolbar/SyncIndicator.svelte"
+    import EditableLabel from "$cmp/EditableLabel.svelte"
 
-    export let title: string
+    export let title: string = "hello"
```
Comment (alexsanjoseph) [inline meiki_ui/src/components/app/toolbar/AppToolbar.svelte]: Why Hello?

Comment (alexsanjoseph) [reply meiki_ui/src/components/app/toolbar/AppToolbar.svelte]: I'm assuming this shouldn't ever be shown, but still something better than hello is  better?

**Review by alexsanjoseph — APPROVED** [review]
LGTM!

File: meiki_ui/src/components/app/toolbar/AppToolbar.svelte (hunk 2 above)
Comment (RainingComputers) [reply meiki_ui/src/components/app/toolbar/AppToolbar.svelte]: yep, will remove

**Pull request merged**